styles.css

```diff
@@ -1,384 +1,377 @@
 /* General */
-
 html {
-    font-family: 'Montserrat', sans-serif;
-  }
+	font-family: 'Montserrat', sans-serif;
+}
 
 body {
-  background-color: #46424D;
-  margin: 0;
-  height: auto;
-  overflow-y: hidden;
+	background-color: #46424D;
+	height: auto;
+	margin: 0;
+	overflow-y: hidden;
 }
 
 main {
-  display: grid;
-  grid-template-columns: 65% 35%;
-  height: 100vh;
-}
-
-img {
-  width: 20px;
+	display: grid;
+	grid-template-columns: 65% 35%;
+	height: 100vh;
 }
 
 button {
```

Comment:
I like the placement of these selectors being at the top of the file vs where they were before